manifests/a/Araxis/Merge/70.30.5981/Araxis.Merge.locale.en-US.yaml

```diff
@@ -0,0 +1,30 @@
+# Created with YamlCreate.ps1 v2.4.1 $debug=NVS0.CRLF.7-4-2.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Araxis.Merge
+PackageVersion: 70.30.5981
+PackageLocale: en-US
+Publisher: Araxis Ltd.
+PublisherUrl: https://www.araxis.com/
+PublisherSupportUrl: https://www.araxis.com/merge/documentation.en
+PrivacyUrl: https://www.araxis.com/legal
+# Author:
+PackageName: Araxis Merge
+PackageUrl: https://www.araxis.com/merge
+License: Proprietary
+LicenseUrl: https://www.araxis.com/merge/windows/licence-agreement.en
+Copyright: Copyright (c) Araxis Ltd.
+CopyrightUrl: https://www.araxis.com/legal
+ShortDescription: Advanced 2 and 3-way file comparison (diff), merging, and folder synchronization
+# Description:
+# Moniker:
+Tags:
+- diff
+- merge
+ReleaseNotes: https://www.araxis.com/merge/windows/release-notes.en
```

Comment:
```suggestion
# ReleaseNotes:
```

Either include the whole text from ReleaseNotesUrl here or comment out this field as URL is not the "release notes".

<img src=https://github.com/microsoft/winget-pkgs/assets/88161975/1e086bb0-4807-47a3-a7d4-29219420b0ad height=300>